setup.cfg

```diff
@@ -1,11 +1,19 @@
 [isort]
-line_length = 120
+indent='    '
+line_length=120
+multi_line_output=3
+known_standard_library=mock,six,time,responses
 known_edx =
 known_django = django
 known_djangoapp = model_utils
 known_first_party = edx_proctoring
+known_third_party=django,rest_framework,ddt,waffle,crum,httmock,edx_rest_framework_extensions,pytz,edx_when,eventtracking,opaque_keys,edx_rest_api_client,webpack_loader,jwt,freezegun,stevedore,Cryptodome
```

Comment:
Curious why you had to add this configuration, given that it's usually the default classification for packages outside the standard library.